config/ignition_fortress_ubuntu_focal.yaml

```diff
@@ -117,24 +210,38 @@ filter_formula: "\
  Package (= libignition-sensors6-segmentation-camera-dev) |\
  Package (= libignition-sensors6-thermal-camera) |\
  Package (= libignition-sensors6-thermal-camera-dev) \
-), $Version (% 6.3.0-2*) |\
-(Package (= ignition-tools) |\
+), $Version (% 6.6.0-3*) |\
+(Package (= gz-tools) |\
+ Package (= ignition-tools) |\
+ Package (= libgz-tools-dev) |\
  Package (= libignition-tools-dev) \
-), $Version (% 1.4.1-1*) |\
+), $Version (% 1.5.0-1*) |\
 (Package (= ignition-transport11) |\
+ Package (= gz-transport11-cli) |\
+ Package (= ignition-transport11-cli) |\
+ Package (= libgz-transport11) |\
+ Package (= libgz-transport11-core-dev) |\
+ Package (= libgz-transport11-dbg) |\
+ Package (= libgz-transport11-dev) |\
+ Package (= libgz-transport11-log) |\
+ Package (= libgz-transport11-log-dev) |\
  Package (= libignition-transport11) |\
  Package (= libignition-transport11-core-dev) |\
  Package (= libignition-transport11-dbg) |\
  Package (= libignition-transport11-dev) |\
  Package (= libignition-transport11-log) |\
  Package (= libignition-transport11-log-dev) \
-), $Version (% 11.0.0-1*) |\
+), $Version (% 11.2.0-3*) |\
 (Package (= ignition-utils1) |\
+ Package (= libgz-utils1) |\
+ Package (= libgz-utils1-cli-dev) |\
+ Package (= libgz-utils1-dbg) |\
+ Package (= libgz-utils1-dev) |\
```

Comment:
According to a fresh run of the generator script, these packages don't yet exist on Focal.
I don't object to assuming they will be eventually but I also see a benefit in waiting for them to appear so that the generator script will help catch any discrepancies between expectation and reality. The import process does not error if an individual package in the filter fails to find a match, which is one of the reasons the generator removes packages from the config which aren't found in the repositories. Between the two processes, we guarantee that exactly the packages available are imported and that each of them has a consistent version and corresponding source package.

In the course of writing all that out, I actually talked myself into removing these so that the generator output matches exactly. That way if it's an "error" that these packages are absent we catch it now. I'm pushing a change now to that effect.

Reply:
We have releasing them, they were missing.